src/components/tx/modals/NftTransferModal/SendNftForm.tsx

```diff
@@ -133,7 +131,7 @@ const SendNftForm = ({ params, onSubmit }: SendNftFormProps) => {
                   error={!!errors.tokenId}
                 >
                   {combinedNfts.map((item) => (
-                    <MenuItem key={item.address + item.id} value={item.id}>
+                    <MenuItem key={item.address + item.id} value={`${item.address};${item.id}`}>
```

Comment:
Nit: we should write a function to create the value to ensure it's the same in the `defaultValues` and here. Might be good to also have a function for getting the selected token as well for consistencies sake.